testRenamePackage.sh

```diff
@@ -4,6 +4,6 @@ cd ..
 rm -rf android-starter-project-copy
 cp -R android-starter-project android-starter-project-copy
 cd android-starter-project-copy
-./renamePackage.sh com.demo.mobile
+python renamePackage.py com.demo.mobile
```

Comment:
I think if we set the executable flag on `renamePackage.py`, we can just change it to `./renamePackage.py ...`.